### ISSUE TITLE
[Feature Request] Integration between storage scanner and crafters

### ISSUE THREAD

**Issue by tjsd** [McJty/RFTools#1400]
Hello! 
I would love to see a functionality where you can see craft-able items from rftools crafters in the storage scanner. Think logisticpipes, it shows up as craft able and you can request the item and it sends the materials needed to the crafter. This could be all connected with xnet cables. 

Keep up the good work!

**Comment by tjsd** [comment]
Will you consider this feature? It would be so cool to be able to get similar functionality from logistic pipes with crafting with patterns, crafters, sattelite pipes and all requestable with storage scanner.

**Comment by McJty** [comment]
It is something that has been on my mind for some time already but I haven't had time to really look into it. So no idea when I will

**Comment by tjsd** [comment]
Do you know Logisticpipes well? Would it help if I made some screenshots of how LP works in 1.7.10 with crafting and stuff? I can make a imgur album with basic functionality and descriptions of the crafting system if you want :)

**Comment by tjsd** [comment]
just a quick bump, wondering if this was something you have started on :)
I guess meecreeps has taken most of your dev time :)

**Comment by McJty** [comment]
No sorry. Havent' had any time for this

**Comment by tjsd** [comment]
Happy New Year! Wow this has dropped to page 3 :) I guess you haven't had time to start looking at this yet? Just a small bump to ask :)

**Comment by McJty** [comment]
This is not that easy and at the moment I'm very busy with other mods and stuff. So I hope you're not waiting for this short term?

**Comment by tjsd** [comment]
No, I'm not holding my breath, just a friendly bump, so it stays in the back of your mind :) With the amount of mods you have and maintain I can understand you being busy!
